.github/MAP_GUIDELINES.md

```diff
@@ -54,3 +54,22 @@ If these features do not work in the current codebase then your map PR will be *
+- **Keep some crime in mind!** Maybe make some areas with less camera coverage, back rooms that are a little odd and musty. Give antags places to play with too.
+- **Windows are important**. Walking through hallways shouldn't feel like a prison. People like to peer in and spy on the 'hard working' department workers, plus, it makes the shape of the map and layout of departments easier to interpret. It's easy to forget when you fly around in ghost mode, but visibility is quite important for hallways. The opposite applies too though, obviously don't make every wall out of glass.
+- **Choke points.** Try not to make entrances to important places like medbay and escape too narrow (unless you're making tiny maps on purpose, of course). It just makes everyone unhappy and then you'll end up engineers removing walls every round then someone pull requests a change-- just save yourself the hassle. In medbay especially, most people get treated right in the lobby or in the main foyer, don't make it too small.
+- Use default maintainence floors under windows. You gotta.
+
+## Tips and Tricks
+###### These aren't one hundred percent necessary to make maps, but they might make your life a bit easier
+- **Cable spawners** `/obj/cablespawner` exist, they autoconnect to neighbouring cables and other cablespawners. It has reinforced and coloured variants and if you want to make it create nodes for terminal connections and whatnot, you can use the `/node` subtype. When needing something pretty and/or handcrafted (like cable art, or two non connected but crossing networks) probably use regular cables placed manually, cablespawners are for convenience in places where you don't care.
+- **Pipe spawners** are similar to cable spawners, but they can't be used to make junctions (you'll have to set directions and priorities manually). Still, they're handy for drawing very long stretches of disposal/mail/cargo/morgue/transport pipe.
+- **Palettes** are small rooms with assortments of walls, floors and such in different arrangements, available for you to copy over to your projects. Palettes are available at https://cdn.discordapp.com/attachments/469379618168897538/1024529074317701211/Palette.dmm 
+- In StrongDMM, "ctrl+1,2,3 to toggle layers, shift click to highlight thing, ctrl shift click to delete thing, 1 2 3 to switch between placing a singular thing, placing multiple things in the area via clickdrag and selecting an area without placing things" ~ Caro, in the #immapper pins
+- At the moment, admins are not really accepting maps unless they offer a big gameplay change (like Nadir). It's good habit to start out by making smaller rooms in order to get used to wiring, APC placement, pipe placement, door permissions and such before tackling a larger project. A lot goes into even small rooms: let's not start out challenging ozymandias, yeah?
```

Comment:
Include, probably after the "big gameplay change (like Nadir)", that they should get dev support as early into the project as possible. We'd rather not have someone work on a whole map to completion with new features and then get told no. Current developer sentiment is against having new maps added unless there's a specific niche or spot that needs filling, which would be requested to be filled if we see a need.